python/test_infra/ttir_builder.py

```diff
@@ -4,25 +4,87 @@
 
 from typing import List, Optional, Union, Tuple
 from ttmlir.ir import *
-from ttmlir.dialects import ttir, tt, func
+from ttmlir.dialects import ttir, tt, func, tensor
 from .ttir.ttir_op_test_base import TTIROpTestBase
 from .tensor_builder import TensorBuilder
+from test_infra.ttir import Add
+import torch
 
 
 class TTIRBuilder:
     def __init__(self, ctx: Context, location: Location):
         self.ctx = ctx
         self.loc = location
+
         tt.register_dialect(self.ctx)
         ttir.register_dialect(self.ctx)
 
         self.tensor_builder = TensorBuilder(self.ctx, self.loc)
 
+    def __wrap_operand_constraints(
```

Comment:
I think python syntax is only one `_`, double `__` is reserved for language constructs.

Reply:
Yeah, dunder methods are special in case of `__method_name__`. I think by convention, since python does not have explicit access  modifiers, one `_` is used for `protected` and double `__` for `private` methods/attributes.

Since `TTIRBuilder` does not have any classes inherit from it, one `_` would also be good. My intention was just to explicitly state that those methods are not meant to be used as public API, that is not their primary purpose.